internal/http/services/owncloud/ocs/handlers/apps/sharing/shares/public.go

```diff
@@ -423,6 +423,13 @@ func (h *Handler) removePublicShare(w http.ResponseWriter, r *http.Request, shar
 }
 
 func ocPublicPermToCs3(permKey int, h *Handler) (*provider.ResourcePermissions, error) {
+
+	if permKey == 31 {
+		// apiSharePublicLink1/createPublicLinkShare.feature:109
+		// Scenario Outline: Trying to create a new public link share of a file with edit permissions only grants read access using the public WebDAV API
```

Comment:
could we instead of having this comment here, link https://github.com/owncloud/ocis/issues/1269 in the changelog? This way we can trace the bug and the fix :D